net/tunneldigger/Makefile

```diff
@@ -1,15 +1,17 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=tunneldigger
-PKG_VERSION:=0.4.4
+PKG_VERSION:=0.3.0
```

Comment:
Uh, we should never downgrade a version. This will make things confused. We should probably bump Tunneldigger version to match this one. Or maybe bump both to 0.5.0.